docs/setup-cluster/deploy-cluster/cluster-configuration.rst

```diff
@@ -0,0 +1,80 @@
+.. _cluster-configuration:
+
+#########################
+ Configuring the Cluster
+#########################
+
+This guide contains basic cluster configuration information and links to reference information.
+
+-  Common configuration reference: :doc:`/reference/deploy/config/common-config-options`
+-  Master configuration reference: :doc:`/reference/deploy/config/master-config-reference`
+-  Agent configuration reference: :doc:`/reference/deploy/config/agent-config-reference`
+
+*********************
+ Basic Configuration
+*********************
+
+The master and agent can each be configured with a configuration file, environment variables, or
+command-line options.
+
+The master and agent both accept an optional ``--config-file`` command-line option, which specifies
+the path of the configuration file to use. Note that when running the master or agent inside a
+container, you will need to make the configuration file accessible inside the container (e.g., via a
+bind mount). For example, this command starts the agent using a configuration file:
+
+.. code::
+
+   docker run \
+     -v `pwd`/agent-config.yaml:/etc/determined/agent-config.yaml \
+     determinedai/determined-agent
+     --config-file /etc/determined/agent-config.yaml
+
+The ``agent-config.yaml`` file might contain
```

Comment:
can we call it `agent.yaml` instead of `agent-config.yaml`?  That's the default name for it.

Reply:
done